tests/frontier/opcodes/test_all_opcodes.py

```diff
@@ -0,0 +1,106 @@
+"""
+Call every possible opcode to see if it is recognized by the evm
+If opcode is undefined the subcall will fail
+"""
+
+from typing import Dict
+
+import pytest
+
+from ethereum_test_forks import Fork
+from ethereum_test_tools import (
+    Account,
+    Address,
+    Alloc,
+    Bytecode,
+    Environment,
+    StateTestFiller,
+    Transaction,
+)
+from ethereum_test_tools.vm.opcode import Opcode
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+from ethereum_test_tools.vm.opcode import UndefinedOpcodes
+
+REFERENCE_SPEC_GIT_PATH = "N/A"
+REFERENCE_SPEC_VERSION = "N/A"
+all_opcodes = set(Op)
+undefined_opcodes = set(UndefinedOpcodes)
+
+
+def prepare_stack(opcode: Opcode) -> Bytecode:
+    """Prepare valid stack for opcode"""
+    if opcode == Op.CREATE:
+        return Op.MSTORE(0, 0x6001600155) + Op.PUSH1(5) + Op.PUSH1(27) + Op.PUSH1(5)
+    if opcode == Op.CREATE2:
+        return Op.MSTORE(0, 0x6001600155) + Op.PUSH1(1) + Op.PUSH1(5) + Op.PUSH1(27) + Op.PUSH1(5)
+    if opcode == Op.JUMPI:
+        return Op.PUSH1(1) + Op.PUSH1(5)
+    if opcode == Op.JUMP:
+        return Op.PUSH1(3)
+    return Op.PUSH1(0x01) * 32
+
+
+def prepare_suffix(opcode: Opcode) -> Bytecode:
+    """Prepare after opcode instructions"""
+    if opcode == Op.JUMPI or opcode == Op.JUMP:
+        return Op.JUMPDEST
+    return Op.STOP
+
+
+@pytest.mark.valid_from("Frontier")
+def test_all_opcodes(state_test: StateTestFiller, pre: Alloc, fork: Fork):
+    """Check if the opcode supported on given fork"""
+    code_worked = 1000
+
+    code_contract: Dict[Opcode, Address] = {}
+    for opcode in sorted(all_opcodes | undefined_opcodes):
+        code_contract[opcode] = pre.deploy_contract(
+            balance=10,
+            code=prepare_stack(opcode) + opcode + prepare_suffix(opcode),
+            storage={},
+        )
+
+    # EVM code to make the call and store the result
+    caller = pre.deploy_contract(
```

Comment:
I know you meant "opcode caller", but on first view `caller` sounded like an EOA to me. How about?
```suggestion
    contract_address = pre.deploy_contract(
```
But feel free to leave as is.